taxi/models.py

```diff
@@ -0,0 +1,18 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(Manufacturer, on_delete=models.CASCADE, related_name="car")
+    drivers = models.ManyToManyField(settings.AUTH_USER_MODEL, related_name="cars")
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=255, unique=True)
```

Comment:
If you want to change the name of model in admin panel, you have to provide `verbose_name` and `verbose_name_plural` in Meta class of model.
https://forum.djangoproject.com/t/django-admin-page-edit-app-names/14720

Reply:
It works, thank you. Can you also explain the reason why the filter field is not displayed in cars?

Reply:
Hey, try to use manufacturer__name for filtering by Manufacturer, this will work for you.